components/brave_ads/core/internal/account/wallet/wallet_info.h

```diff
@@ -15,6 +15,7 @@ struct WalletInfo final {
 
   bool WasUpdated(const WalletInfo& other) const;
   bool HasChanged(const WalletInfo& other) const;
+  bool WasCreated(const WalletInfo& other) const;
```

Comment:
Nit: could we move to before WasUpdated as this would mean the order is created, updated and then changed